### DISCUSSION TITLE
Save Layouts across Multiple Desktops and Multiple Screens

### DISCUSSION THREAD

**Discussion by sissnad** [rxhanson/RectanglePro-Community#215] [Ideas]
I would love to be able to save layouts across desktops and even across multiple screens. For example my typical workflow switches between mac book screen only and one external monitor & mac book screen. I use multiple desktops on both screens. I would like to save the layout I use with

1. mac book screen across multiple desktops
2. macbook screen with multiple desktops and external screen with multiple desktops

Bonus:  Rectangle automatically switches between layout 1 and 2 when it detects that I connect/disconnect external screen.

**Comment by rxhanson** [comment]
Thanks for the feedback. Unfortunately, macOS does not provide a supported way to work with spaces. Some apps move windows between spaces, but they are using kind of hacks to do it. I only like to add in things I can guarantee will work for everyone, so it's dependent on when Apple adds a supported way.

**Comment by sissnad** [comment]
Thanks for the reply. Fully get your point. And I think I know what you mean. Have been trying to find a solution via apple scripts, but only found ways that were close to macro recording mouse clicks 😅

Love rectangle though, great app!
